src/tools/stereoCalib/src/stereoCalibThread.cpp

```diff
@@ -43,6 +44,30 @@ stereoCalibThread::stereoCalibThread(ResourceFinder &rf, Port* commPort, const c
 
 bool stereoCalibThread::threadInit()
 {
+/* FOR DEBUG LOAD IMAGES FROM FILE
```

Comment:
Please, get rid of all these comments.

Reply:
Yes, sorry about that.

Reply:
Still need to push?

Reply:
Pleas check the latest commit a2692d30bf9e5da41122c95dc45bbdd13e978713